header.php

```diff
@@ -63,13 +63,13 @@
 	<header class="header" role="banner">
 		<div class="header__inside">
 			<div class="header__brand">
+				<?php
+				$root_id = get_network()->site_id;
+				switch_to_blog( $root_id );
+				?>
 				<a aria-label="<?php echo get_bloginfo( 'name', 'display' ); ?>" href="<?php echo network_home_url(); ?>">
```

Comment:
get_bloginfo will now return the name of the root site, instead of the current book/site